src/metabase/api/notification.clj

```diff
@@ -86,8 +86,11 @@
 
 (defn- send-you-were-added-card-notification-email! [notification]
   (when (email/email-configured?)
-    (messages/send-you-were-added-card-notification-email!
-     (update notification :payload t2/hydrate :card) (all-email-recipients notification) @api/*current-user*)))
+    (when-let [recipients-except-creator (->> (all-email-recipients notification)
+                                              (remove #(= (:email @api/*current-user*) %))
```

Comment:
nit: I prefer using a set-as-predicate instead of a function for this kind of case, because its a bit safer from a performance perspective.

In this style you're resolving a dynamic var and doing. map look-up for every recipient. Probably fine for the sizes we're dealing with, but I think life is easier to just use one style. Because I always use sets, I've gotten to find it easier to read also.

Reply:
+1 here, it's little, but this constant access to a dynamic var is bothering me :)

Reply:
This is the only thing blocking me from approving 😄

Reply:
performance boost 🏎️ a2e61c2c47